src/index.js

```diff
@@ -0,0 +1,74 @@
+import './style.css';
+import addDoList from './modules/AddList.js';
+import deleteList from './modules/DeleteList.js';
+import {
+  getList, addList, removeList, updateList,
+} from './modules/UpdateList.js';
+import completedList from './modules/CompletedList.js';
+
+const doList = () => {
+  const todos = getList() || [];
+  if (todos) {
+    todos.map((task) => addDoList(task));
+  }
+};
+doList();
+
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const todos = getList();
+  const toAdd = document.getElementById('addTasks').value;
+  const todoTask = {
+    index: todos.length,
+    description: toAdd,
+    completed: false,
+  };
+
+  if (toAdd !== '') {
```

Comment:
It is best practice to use conditional short hands. Use this approach only for Boolean values and if you are sure that the value will not be undefined or null. for this case, you can change it to 'if(todo)'.